tower/src/util/mod.rs

```diff
@@ -938,3 +940,32 @@ pub trait ServiceExt<Request>: tower_service::Service<Request> {
 }
 
 impl<T: ?Sized, Request> ServiceExt<Request> for T where T: tower_service::Service<Request> {}
+
+/// Convert an `Option<Layer>` into a [`Layer`].
+///
+/// ```
+/// # use std::time::Duration;
+/// # use tower::Service;
+/// # use tower::builder::ServiceBuilder;
+/// # use tower::util::option_layer;
```

Comment:
TIOLI: in general, I think we do tend to show the type or function in question being imported, in these examples:
```suggestion
/// use tower::util::option_layer;
```